Python 3.8 benchmarks failed!
